README.md

```diff
@@ -18,15 +20,15 @@
 
 # Get started
 The best way to get your hands on.  
-* NodeJS [demo](https://github.com/AbsaOSS/libvcx/tree/master/wrappers/node) - NodeJS wrapper is our first class citizien 
+* NodeJS [demo](https://github.com/hyperledger/aries-framework-rs/tree/master/wrappers/node) - NodeJS wrapper is our first class citizien 
```

Comment:
I think we should choose more specific and accurate formulations in the docs - something like 'NodeJS wrapper is currently the most up to date wrapper', or 'NodeJS wrapper is currently our first priority'. Alternatively, leave this out completely and only mention that the Python wrapper has been unmaintained.  Secondly, `citizien` sounds like a failed attempt at French :)